changelogs/unreleased/8803-add-consts-to-stable-api.yml

```diff
@@ -0,0 +1,15 @@
+change-type: patch
+description: >
+  Adds the following consts to the stable api:
+    - UNDEPLOYABLE_STATES
+    - UNDEPLOYABLE_NAMES
+    - TRANSIENT_STATES
+    - NOT_DONE_STATES
+    - DONE_STATES
+    - INITIAL_STATES
+    - TERMINAL_STATES
+    - VALID_STATES_ON_UPDATE
+
+destination-branches: [master]
+sections:
+  other-note: "{{description}}"
```

Comment:
I feel like this should be visible to the end user but not sure in which section it falls under